nemo_skills/inference/retrieve_similar.py

```diff
@@ -63,7 +77,10 @@ class RetrieveSimilarConfig:
     top_k: int = 3
     retrieve_key: str = 'problem'
 
-    batch_size: int = 512
+    batch_size: int = 2048
```

Comment:
Add a line to describe batch_size so that reader can easily distinguish it from chunk_size